dashboard/src/shared/ServiceBinding.ts

```diff
@@ -52,6 +41,11 @@ export interface IServiceBinding {
   };
 }
 
+export interface IServiceBindingWithSecret {
```

Comment:
if we are going to be using `IServiceBindingWithSecret` always: does it worth to expose the current `IServiceBinding`? I think it would be less confusing to just export one so it's easier to work with this definition. (Also if only one is exported I would name it `IServiceBinding`).

Reply:
You're right, I'm not sure why I decided to keep some props using both. I'll change that.